src/core/server/http/integration_tests/request.test.ts

```diff
@@ -180,24 +187,25 @@ describe('OpenSearchDashboardsRequest', () => {
 
   describe('events', () => {
     describe('aborted$', () => {
-      it('emits once and completes when request aborted', async (done) => {
+      // Flaky test
+      it.skip('emits once and completes when request aborted', async () => {
```

Comment:
are these flaky only in the CI? I was able to get the passing locally.

Reply:
+1

Reply:
When you pulled the PR down it consistently passed locally for you? Sometimes it would pass, sometimes it would fail for me.

Reply:
Could you please elaborate on the failure?

Reply:
> When you pulled the PR down it consistently passed locally for you? Sometimes it would pass, sometimes it would fail for me.

For main these tests pass consistently. Let try with removing these skips.

Reply:
Yeah they've been passing consistently, but not after this upgrade.

Reply:
I see, I believe we have to make an update still: https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1301/files#r818949420

Reply:
I still couldn't get this one to pass. This test doesn't use fake timers.

Reply:
This might be related to the `supertest` functions used.

Reply:
> This might be related to the `supertest` functions used, but I haven't narrowed it down.

If you aren't able to for this PR could we create a follow up issue to look into it? I don't want to block this PR any longer because it prevents an improvement that brings in more value than this test can provide but I don't want to forget about this test.